Essentials/AutoCommands.cs

```diff
@@ -23,11 +29,59 @@ public void Start()
             _timer.Start();
         }
 
+        public void RunOnStart()
+        {
+            foreach (var command in EssentialsPlugin.Instance.Config.AutoCommands)
+            {
+                if (command.CommandTrigger != Trigger.OnStart)
+                    return;
+                else if(command.CommandTrigger == Trigger.OnStart)
+                {
+                    if(TimeSpan.Parse(command.Interval) == ((ITorchServer)TorchBase.Instance).ElapsedPlayTime)
```

Comment:
This will almost never evaluate true. You need to compare with an epsilon value. e.g.:
`if((a - b).TotalSeconds < 1 && (a - b).TotalSeconds > 0)`

Reply:
I can do that, but will need to use <= 1 due to that method running from the TimerElapser with the 1sec timer.